### ISSUE TITLE
Exception Occurred when Pulling File

### ISSUE THREAD

**Issue by marshl** [marshl/Lobster#88]
```
2016-05-11 14:35:24.098 [INFO]: Connecting to database PharmCIS Fox5 Dev2
2016-05-11 14:35:24.933 [ERROR]: Unhandled Exception System.ArgumentNullException: Value cannot be null.
Parameter name: path
   at System.IO.Path.GetFullPath(String path)
   at LobsterModel.DatabaseConnection.GetClobDirectoryForFile(String fullpath) in C:\Users\Fivium\Documents\Lobster\LobsterModel\DatabaseConnection.cs:line 152
   at LobsterModel.Model.SendDownloadClobDataToFileMessage(DatabaseConnection databaseConnection, String fullpath) in C:\Users\Fivium\Documents\Lobster\LobsterModel\Model.cs:line 175
   at LobsterWpf.MainWindow.PullButton_Click(Object sender, RoutedEventArgs e) in C:\Users\Fivium\Documents\Lobster\LobsterWpf\MainWindow.xaml.cs:line 397
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.Controls.Button.OnClick()
   at System.Windows.Controls.Primitives.ButtonBase.OnMouseLeftButtonUp(MouseButtonEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at LobsterWpf.App.Main() in C:\Users\Fivium\Documents\Lobster\LobsterWpf\obj\Release\App.g.cs:line 0
```

**Comment by marshl** [comment]
Only occurs when attempting to Pull files in "database mode"

**Comment by marshl** [comment]
Fixed with edc0cb4
